README.md

```diff
@@ -1,6 +1,30 @@
 # GTFS to GeoJson
 
-This Rust crate is used to convert GTFS data to the GeoJSON format.
+This Rust crate is used to convert GTFS data to the GeoJSON format. 
+
+## Installing Crate
+
+[Crate](https://crates.io/crates/gtfs-geojson)
+
+While the repo is named `gtfs-to-geojson` the crate is called `gtfs_geojson` and can me installed using 
+
+```bash
+cargo add gtfs_geojson
+```
+
+## Functions
+
+The `gtfs_geojson` library use the `gtfs_structures` library and there types to convert gtfs files over to geojson.
```

Comment:
```suggestion
The `gtfs_geojson` library use the `gtfs_structures` library and its types to convert gtfs files over to geojson.
```

Reply:
Sure thing! I love transit and work with GTFS a lot and I had built [gtfs-parser](https://github.com/AvidDabbler/gtfs-parser) recently, which is a project that is similar, but in typescript inspired by the out of date [gtfs2geojson](https://github.com/node-geojson/gtfs2geojson). I was looking for a drop in solution for a project called [simple-transit-map](https://github.com/ioTransit/simple-transit-map). 

Eventually I would like to use rust on client side alongside mapbox and maybe extend one what i have built with simple-transit-map to ingest gtfs directly on client.